hack/cve-triage/README.md

```
@@ -0,0 +1,277 @@
+
+# Apply (posts comment + closes ticket)
+python3 hack/cve-triage/triage_operator_cve.py OCPBUGS-XXXXX
+
+# Also accepts full Jira URL
+python3 hack/cve-triage/triage_operator_cve.py \
+    "https://redhat.atlassian.net/browse/OCPBUGS-XXXXX"
+```
+
+### Workflow (8 steps)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)